pages/learn/deploy/deployment.md

```diff
@@ -15,9 +15,9 @@ __Note:__ The resin.io git repository is not intended as a code hosting solution
 
 ## The build server
 
-The build server is an incredibly powerful tool which cross-compiles code for the target device on our (far more powerful) server. This gives you the ability to compile a gnarly dependency tree in seconds on the server rather than minutes or even hours to build on your device.
+A build server is a powerful tool which compiles code specifically for your device's architecture. With our build servers, compiling a gnarly dependency tree can be done in seconds, as compared to the minutes or even hours it may take to build on your device.
```

Comment:
I don't really love `A build server...`, preferred the `The build server...` or maybe `Our build servers...`.
`A build server` sounds like describing any build server while we later provide specific characteristics like `is a powerful tool` & `your device's arch`.